bin/templates/cordova/lib/builders/AntBuilder.js

```diff
@@ -107,7 +107,18 @@ AntBuilder.prototype.build = function(opts) {
     var args = this.getArgs(opts.buildType == 'debug' ? 'debug' : 'release', opts);
     return check_reqs.check_ant()
     .then(function() {
-        return spawn('ant', args, {stdio: 'inherit'});
+        return spawn('ant', args, {stdio: 'pipe'});
+    }).progress(function (stdio){
```

Comment:
Can you elaborate? As far as I can see superspawn only returns `stderr` output only if you 'pipe'. Piping implies that someone has to still write this to process.stdout & process.stderr.